test/unit/collection-view.empty-view.spec.js

```diff
@@ -279,7 +279,7 @@ describe('collectionview - emptyView', function() {
       });
 
       it('renders other empty view instance', function() {
-        expect(this.getEmptyViewStub).to.have.been.calledWith(this.collectionView.children.first());
+        expect(this.getEmptyViewStub).to.have.been.called;
```

Comment:
Ok I can verify.. I pulled a fresh `next` and updated the chai's and mocha's and sinon's.. I'm not sure which one it is for certain, but updating them with no other changes will fail this test:
```
AssertionError: expected getEmptyView to have been called with arguments undefined
      getEmptyView() at [object Object].Marionette.CollectionView.Marionette.AbstractView.extend.showEmptyView (src/collection-view.js:348:26)
      at Context.<anonymous> (test/unit/collection-view.empty-view.spec.js:282:52)
```